internal/kubernetes/status/status.go

```diff
@@ -394,7 +394,7 @@ func findIssuers(ctx context.Context, cfg *rest.Config) ([]summaryIssuer, error)
 			}
 			for _, issuer := range issuers.Items {
 				summaryIssuers = append(summaryIssuers, summaryIssuer{
-					APIVersion: kmsissuerv1alpha1.GroupVersion.String(),
+					APIVersion: veiv1alpha1.SchemeGroupVersion.Group,
```

Comment:
@irbekrm why is this just the group?

Reply:
Thanks for spotting this @inteon ! I can't remember any reason for using just the group, so most likely it was a typo.

Reply:
Fixed in https://github.com/jetstack/jsctl/pull/72/commits/25fdd77b7bcfce03f3b05c1b4c5fe9828d5df1d9